### PR TITLE
[AND-82] Implement bottom navigation entry for updates

### PR DESCRIPTION
**Pull request by pedroagostinhoaptoide**
**What does this PR do?**

~**IN DRAFT WHILE THE REQUIRED STRINGS ARE NOT ADDED**~

   - Adds a bottom navigation item for the updates
   - Creates the skeleton screen for the updates.
   - Fixes an issue in the bottom bar items alignment, that started after updating the compose version.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] See by commit.

**How should this be manually tested?**

  Flow on how to test this or QA Tickets related to this use-case: [AND-82](https://aptoide.atlassian.net/browse/AND-82)

**What are the relevant tickets?**

  Tickets related to this pull-request: [AND-82](https://aptoide.atlassian.net/browse/AND-82)

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

[AND-82]: https://aptoide.atlassian.net/browse/AND-82?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[AND-82]: https://aptoide.atlassian.net/browse/AND-82?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by phillipsgoncalves — APPROVED** [review]
LGTM, just missing the translations like you said.

File: app-games/src/main/java/com/aptoide/android/aptoidegames/updates/UpdatesScreen.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  UpdatesScreen(
+    navigate = navigate,
+    navigateBack = navigateBack
+  )
+}
+
+@Composable
+fun UpdatesScreen(
+  navigate: (String) -> Unit,
+  navigateBack: () -> Unit,
```
Comment (Culurio) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/updates/UpdatesScreen.kt:25]: Do we need the navigate back to this screen?

Comment (pedroagostinhoaptoide) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/updates/UpdatesScreen.kt]: Fixed, removed navigateBack

**Review by GuiFPires — APPROVED** [review]

File: app-games/src/main/java/com/aptoide/android/aptoidegames/home/BottomBarMenus.kt [hunk 2]
```diff
@@ -1,38 +1,82 @@
...
 import com.aptoide.android.aptoidegames.drawables.icons.getSearch
 import com.aptoide.android.aptoidegames.search.presentation.searchRoute
 import com.aptoide.android.aptoidegames.theme.Palette
+import com.aptoide.android.aptoidegames.updates.updatesRoute
 
 sealed class BottomBarMenus(
   val route: String,
   val titleId: Int,
-  val icon: ImageVector,
+  val icon: @Composable (() -> Unit),
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/home/BottomBarMenus.kt:33]: Let's not mix composable functions with enumerations definitions like sealed classes

**Review by G0retZ — CHANGES_REQUESTED** [review]

File: app-games/src/main/java/com/aptoide/android/aptoidegames/home/BottomBarMenus.kt (hunk 2 above)
Comment (pedroagostinhoaptoide) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/home/BottomBarMenus.kt]: Fixed

**Review by G0retZ — APPROVED** [review]

**Pull request merged**